### DISCUSSION TITLE
How do you set an image to display in GitHub repository search results?

### DISCUSSION THREAD

**Discussion by Micsupreeme** [community/community#23503] [New to GitHub]
<p>Hi GitHub Community,</p>
<p>I like searching repositories by tags and I occasionally see repositories in the search results with images above their names (as pictured and pointed out by the orange box)</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/4/f/4f73bf47c3e5b57c3ea0eed95840255b9283c073.png" data-download-href="/uploads/short-url/bkRHR8YA4CNgPaqva4AyRgTtNAL.png?dl=1" title="repo_image_in_github_search_results" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/19594864/181077452-0308973c-12bc-477e-8afe-f833a3747812.png" alt="repo_image_in_github_search_results" data-base62-sha1="bkRHR8YA4CNgPaqva4AyRgTtNAL" width="525" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/4/f/4f73bf47c3e5b57c3ea0eed95840255b9283c073_2_525x500.png, https://aws1.discourse-cdn.com/github/optimized/3X/4/f/4f73bf47c3e5b57c3ea0eed95840255b9283c073_2_787x750.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/4/f/4f73bf47c3e5b57c3ea0eed95840255b9283c073.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/4/f/4f73bf47c3e5b57c3ea0eed95840255b9283c073_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">repo_image_in_github_search_results</span><span class="informations">857×815 63.3 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p>What determines whether or not an image displays here? Is there a popularity/star requirement? Is it actually derrived from the README? Or is it just a hidden repo setting I haven’t found yet?</p>
<p>Thanks in advance!</p>

**Comment by jfullstackdev** [comment, marked as answer]
<p>you mean the Social Preview at Settings ?</p>
<p><img src="https://user-images.githubusercontent.com/47092464/181077453-2ed7b5c5-3a10-4c58-882e-61fd8f7a9c7c.png" alt="image" data-base62-sha1="iemIxKLVxdfDRwrQhHoLQw1oVLu" width="687" height="468"></p>
<p>in this example, it’s blank, you need to upload that preview</p>

**Reply by Manishmg3994** [reply]
Yes , but this time I want to know how we can set this with GitHub Actions say workflow using python
